java/src/main/java/lesson02/solution/HelloActive.java

```diff
@@ -45,6 +45,5 @@ public static void main(String[] args) {
         String helloTo = args[0];
         Tracer tracer = Tracing.init("hello-world");
         new HelloActive(tracer).sayHello(helloTo);
-        tracer.close();
```

Comment:
I have been waiting for https://github.com/opentracing/opentracing-java/issues/250 and https://github.com/opentracing/specification/pull/128 to move forward with the Closeable spec, but it seems this will take a while. I've updated this pull request to upcast to JaegerTracer in a try-resource block.